uwblueprint/supportive-housing
# Create Log Record Frontend Component

## Notion ticket link

[Log Record Frontend Component](https://github.com/uwblueprint/supportive-housing/issues/29)



## Implementation description
* Created a new page, `LogRecords.tsx` that displays logs in a table.
* Created new endpoint, `/log-records` that will open up the page.
* Styled using lo-fi for font, font colour, font size, etc.



## Steps to test
1. Verify that going to the `/log-records` opens up the Log Records page. 
2. Verify that all information displays correctly, and that rows grow vertically if the text does not fit horizontally (refer to the fourth row for this).
3. Verify that styling matches the styling on the Figma lo-fis.



## What should reviewers focus on?
* Are there any styling changes that should be made?


## Checklist
